packages/rum/src/domain/record/shadowRootsController.ts

```diff
@@ -57,6 +57,8 @@ export const initShadowRootsController = (
         addTelemetryDebug('no shadow root in map', {
           shadowRoot: shadowRoot ? shadowRoot.nodeName : 'no node name',
           childrenLength: shadowRoot ? shadowRoot.childElementCount : '-1',
+          controllerByShadowRootSize: controllerByShadowRoot.size,
+          html: shadowRoot && isExperimentalFeatureEnabled('shadow_dom_debug') ? shadowRoot.innerHTML : undefined,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏as discussed together, you could add `view.name` here, if it can help you detect some pattern

Reply:
How can I get that here?

Reply:
Getting the view.name might be non-trivial here (either passing the ViewContext or tracking the current view with LifeCycle events could be a bit overkill for such investigation)

I shared with you a way to make it easier to navigate to the related view for telemetry events, maybe it could be enough?

Reply:
a dirty way (for temporary debug sake) could be to access it from the `DD_RUM.getInternalContext().view.name` 🙈

Reply:
honestly I can do without, I would rather merge this PR (on Monday) in its current state than trying to get the perfect attributes